module.rb

```diff
@@ -0,0 +1,27 @@
+module Country_status
+  
+  def country_will_get_loan(gdp, state)
+    if gdp > 300 && state == "stable"
+      "YES"
+    else
+      "NO"
+    end
```

Comment:
```suggestion
    gdp > 300 && state == "stable" ? "Yes" : "No"
```
Make all methods in that format